lib/config/pubspec_config.dart

```diff
@@ -37,6 +38,7 @@ class PubspecConfig {
       useFvm = false;
       configPath = _DEFAULT_CONFIG_PATH;
       equalsHashCode = false;
+      explicitToJson = false;
```

Comment:
Not the same as the default specified in docs

Reply:
fixed